crate/kmip/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license-file = "../../LICENSE.md"
 doctest = false
 
 [features]
-default = []
+default = ["dep:sha3"]
 openssl = ["dep:openssl"]
```

Comment:
Shouldn't be openssl in default features at this point ?

Reply:
I don't think so, Openssl comes with heavy dependencies regarding the system, and one could want to avoid those deps.

As an example, the CLI and Client crates does not need openssl, and it would be weird to require them.